content/source/docs/enterprise/private/faq.html.md

```diff
@@ -25,6 +25,8 @@ This page will provide answers to many common questions around Private Terraform
 
 ## About the Private Terraform Enterprise AMI
```

Comment:
On this page, let's take the time to reorganize so that there is a subsection for the AMI and only that section has the deprecation, especially since not all the sections on this page are valid anymore (😖). I'm thinking something like this for the organization, and a similar TOC:

```
# Private Terraform Enterprise Frequently Asked Questions

## Managing Tool Versions
## Migration from SaaS-based Terraform Enterprise
## Support for Private Terraform Enterprise

## AMI

{deprecation notice}

### About the AMI
### AMI IDs
### Additional Configuration Info
### Network Access
### Storing Terraform Enterprise State
### Private Terraform Enterprise Architecture
### Advanced Terraform
### Rekeying Vault
### Minimum Viable IAM Access Policy
```

Reply:
this is a good suggestion, ty!